### DISCUSSION TITLE
CSV Import Site -scoped VLANGroups

### DISCUSSION THREAD

**Discussion by sdktr** [netbox-community/netbox#6897] [Q&A] [answered]
Hi netboxians,

I'm confused about the format we should use for importing old-skool site-scoped VLANGroups. According to the table on the import page:
![image](https://user-images.githubusercontent.com/19833687/128379743-37228beb-305a-43e4-b5e7-0479bcb6c915.png)

However when using the below import we end up with no Site assignment
```
name,slug,site
DM-NYC,DM-NYC,DM-NYC
```

Adding a scope_type header, pointing to one of the [accepted types](https://github.com/netbox-community/netbox/blob/15ed5752078d186ec7dcb8a457ae72de1788e885/netbox/ipam/constants.py#L62) renders an error on import:
```
name,slug,site,scope_type
DM-NYC,DM-NYC,DM-NYC,site
```

> Row 1 scope_type: Select a valid choice. That choice is not one of the available choices.

Any clues how a working import should look like? BR Stefan

**Comment by candlerb** [comment, marked as answer]
This is a duplicate of #6691, but the poster there didn't follow up with a bug report.

I suggest you raise a bug for the CSV bulk import of VLANGroups, noting that:
- setting "scope_type" to "site" is rejected as an invalid value
- a "site" column is offered, but the values are ignored

I suspect both problems arise from [here](https://github.com/netbox-community/netbox/blob/v2.11.10/netbox/ipam/forms.py#L1240-L1251) which wasn't updated.

**Reply by sdktr** [reply]
Followed up with https://github.com/netbox-community/netbox/issues/6909 and https://github.com/netbox-community/netbox/issues/6908
